src/components/Checkbox/Checkbox.tsx

```diff
@@ -72,40 +75,53 @@ export class Checkbox extends React.Component<CheckboxProps> {
 
   render() {
     const {
+      box,
       checked,
       disabled,
       label,
       error,
       indeterminate,
       onChange,
       name,
+      suffix,
       className,
     } = this.props;
     const iconContent = this._renderIcon();
 
     return (
-      <CoreCheckbox
-        className={st(classes.root, { checked, disabled, error }, className)}
-        {...this.getDataAttributes()}
-        data-hook={this.props['data-hook']}
-        checkedIcon={iconContent}
-        uncheckedIcon={iconContent}
-        indeterminateIcon={iconContent}
-        indeterminate={indeterminate}
-        checked={checked}
-        onChange={onChange}
-        name={name}
+      <div
+        className={`${box && classes.box} ${checked &&
+          classes.checked} ${disabled && classes.disabled} `}
```

Comment:
In Stylable, the root element should receive the `.root` class.
and Stylable states should be used instead of regular classes (see the `root` below).
So since the dom structure has changed here, the stylesheet structure should change as well
(and again, I would like you to talk to @mariakwix and figure out how to solve this together, since the same solution should appear in the `RadioButton` component she's working on - the best approach would be to have a single abstraction/component with these themes, so that both `RadioButton` and `Checkbox` could use it without code duplication)

Reply:
Note that this DOM change, also caused the `Checkbox`'s tests to fail (you can see in the status check at the bottom of the PR page)

Reply:
While it's possible to share some CSS between the RadioButton's and Checbox's box, I'm not sure it will be worth the added complexity of passing the state along as arguments (selected, disabled, hovered). There isn't any logic in the box, it's just some CSS.